IBAnimatable/FoldAnimator.swift

```diff
@@ -39,22 +39,12 @@ public class FoldAnimator: NSObject, AnimatedTransitioning {
       self.folds = folds
     }
 
-    switch fromDirection {
-    case .right:
-      self.transitionAnimationType = .fold(from: .right, folds: folds)
-      self.reverseAnimationType = .fold(from: .left, folds: folds)
-      self.interactiveGestureType = .pan(from: .left)
-      reverse = true
-    case .top:
-      self.transitionAnimationType = .fold(from: .top, folds: folds)
-      self.reverseAnimationType = .fold(from: .bottom, folds: folds)
-      self.interactiveGestureType = .pan(from: .bottom)
-      reverse = false
-    case .bottom:
-      self.transitionAnimationType = .fold(from: .bottom, folds: folds)
-      self.reverseAnimationType = .fold(from: .top, folds: folds)
-      self.interactiveGestureType = .pan(from: .top)
-      reverse = true
+    switch direction {
```

Comment:
I think we don't even need the `switch` here, the `default` case is pretty much `.left`.